src/Service/PaymentStateDataService.php

```diff
@@ -109,30 +109,6 @@ public function getPaymentStateDataFromContextToken(string $contextToken): ?Paym
         return $stateDataRow;
     }
 
-    /**
-     * @param string $contextToken
-     * @return string|null
-     */
-    public function getPaymentMethodType(string $contextToken): ?string
```

Comment:
Being a public function should we deprecate it until next major release instead?

Reply:
Good point